.github/ISSUE_TEMPLATE/NEW-FEATURE-REQUEST.yml

```diff
@@ -0,0 +1,43 @@
+name: 🌟 New Feature request
+description: Request a new feature for Skin
+title: "[NEW FEATURE] <title>"
+labels: ["type: feature", "Needs Triage"]
+body:
+- type: textarea
+  id: feature-description
+  attributes:
+    label: Describe the feature/module in detail
```

Comment:
These labels are a little inconsistent in tone. I see these two are `Describe the feature/module in detail` and `Screenshots`. They should either be
```
Description
Screenshots
```
or
```
Describe the feature/module in detail
Share screenshots of the feature/module
```
My vote is for the shorter descriptions, but it's up to you. This also applies across the board on both of these templates.

Reply:
I'll revise this one to be consistent, but in some cases, I prefer to opt to be more descriptive than brief at the expense of losing clarity.